VERSION

```diff
@@ -0,0 +1 @@
+0.31.0
```

Comment:
It looks like the latest version is `0.3.1`. Did you mean to enter that? If so, you'll want to enter 0.3.2 since Expeditor will a) assume if you modify the version file you are putting the right version and b) doesn't like trying to create tags that already exist.